torch/csrc/utils/pybind.h

```diff
@@ -5,6 +5,7 @@
 #include <ATen/ATen.h>
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
+#include <pybind11/operators.h>
```

Comment:
nit: this was probably added accidentally?

Reply:
Again it's intentional... It's necessary for the `py::self` operator overloading syntax.